src/Kirel.Identity.Server.Domain/UserConfig.cs

```diff
@@ -0,0 +1,38 @@
+namespace Kirel.Identity.Server.Domain
+{
+    /// <summary>
+    /// Configuration class for users
+    /// </summary>
+    public class UserConfig
+    {
+        /// <summary>
+        /// Gets or sets the username.
+        /// </summary>
+        public string? UserName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the email address.
+        /// </summary>
+        public string? Email { get; set; }
+
+        /// <summary>
+        /// Gets or sets the user's first name.
+        /// </summary>
+        public string? Name { get; set; }
```

Comment:
same, use a public string Name { get; set; } = ""; if you want to suppress warnings.